src/components/canrawview/canrawview_p.h

```diff
@@ -44,6 +50,31 @@ class CanRawViewPrivate : public QWidget {
 
     ~CanRawViewPrivate() {}
 
+    void saveSettings(QJsonObject& json)
+    {
+        QJsonObject jObjects;
+        QJsonArray viewModelsArray;
+        /*
+        QString fileName("viewSave.cds");
+        QFile saveFile(fileName);
+
+        if (saveFile.open(QIODevice::WriteOnly) == false) {
+            cds_debug("Problem with open the file {0} to write confiruration", fileName.toStdString());
+            return;
+        }
+        */
+        writeColumnsOrder(jObjects);
+        jObjects["Sorting"] = prevIndex;
+        jObjects["Scrolling"] = (ui->freezeBox->isChecked() == true) ? 1 : 0;
```

Comment:
I would put constant on the left, i.e. `(true == ui->freezeBox->isChecked()) ? 1 : 0;`, or just: `(ui->freezeBox->isChecked() ? 1 : 0)`.

Additionally, `assert(nullptr != ui);`, `assert(nullptr != ui->freezeBox);`.